packages/firebase/README.md

```diff
@@ -83,7 +83,7 @@ var app = firebase.initializeApp({ ... });
 If you are using ES6 imports or TypeScript:
 
 ```
-import firebase from 'firebase';
```

Comment:
I think there's another instance (`import firebase from 'firebase/app'`) that should be updated too?

Reply:
good catch! done.